domain/api.go

```diff
@@ -312,3 +323,67 @@ func CancelDomainAction(c gotransip.Client, domain string) error {
 
 	return c.Call(sr, nil)
 }
+
+// RequestAuthCode requests the authcode at the registry
+func RequestAuthCode(c gotransip.Client, domainName string) (string, error) {
+	sr := gotransip.SoapRequest{
+		Service: serviceName,
+		Method:  "requestAuthCode",
+	}
+	sr.AddArgument("domain", domainName)
+
+	var v string
+	err := c.Call(sr, &v)
```

Comment:
What happens if the function returns "null" rather than a string? See docs: https://api.transip.nl/docs/transip.nl/class-Transip_DomainService.html#_requestAuthCode

Reply:
an empty string is parsed from the XML response